seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/serializable/TaskDataSerializerHook.java

```diff
@@ -148,6 +149,8 @@ public IdentifiedDataSerializable create(int typeId) {
                     return new CleanTaskGroupContextOperation();
                 case SOURCE_READER_EVENT_OPERATOR:
                     return new SourceReaderEventOperation();
+                case CHECK_TASKGROUP_IS_EXECUTING:
```

Comment:
Any Code use this? I can't find it.

Reply:
https://github.com/apache/incubator-seatunnel/blob/ac4e880fb58ece7e0920c962c11f4d768d6519b1/seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/dag/physical/PhysicalVertex.java#L217

If don't add this, that will report an error.